src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2496,6 +2510,13 @@ private static unsafe void FreeArray(byte** arr, int length)
             int redirectStdin, int redirectStdout, int redirectStderr, int creationFlags,
             out int lpChildPid, out int stdinFd, out int stdoutFd, out int stderrFd);
 
+        [DllImport("libpsl-native")]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static extern bool KillProcess(int pid);
+
+        [DllImport("libpsl-native")]
+        internal static extern int WaitPid(int pid, bool nohang);
+
```

Comment:
I'd move this in https://github.com/PowerShell/PowerShell/blob/5abb90f21a12e1816312fb3b204835a2ad9d1065/src/System.Management.Automation/CoreCLR/CorePsPlatform.cs#L1043